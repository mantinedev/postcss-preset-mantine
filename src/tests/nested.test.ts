import { testTransform } from './utils';

const baseInput = `
.a {
    .b {
        color: black;
    }
}
`;

describe('nested', () => {
  it('it transforms nested rules correctly', async () => {
    const res = await testTransform(baseInput);
    expect(res.css).toMatchSnapshot();
  });
});
