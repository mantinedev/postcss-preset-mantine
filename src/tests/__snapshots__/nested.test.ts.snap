// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nested it transforms nested rules correctly 1`] = `
"
.a .b {
        color: black;
    }
"
`;
